### DISCUSSION TITLE
queued workflows runs are failing to be cancelled

### DISCUSSION THREAD

**Discussion by urit** [community/community#110798] [Actions]
### Select Topic Area

Bug

### Body

can't cancel the queued workflow run ....

when running 
`gh run cancel 999999999`
getting
```
HTTP 500: Failed to cancel workflow run (https://api.github.com/repos/XXXX/XXX/actions/runs/999999999/cancel)
```

**Comment by urit** [comment]
<img width="1497" alt="cancel" src="https://github.com/community/community/assets/454325/4c226669-6309-4bfc-89d1-b9ec66aadf28">

**Reply by ArthurKun21** [reply]
Have you solved this?
